src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/GeneralTests.cs

```diff
@@ -286,6 +286,26 @@ public static void RoundTrip_ExplicitSki()
             Assert.Equal<byte>(contentInfo.Content, ecms.ContentInfo.Content);
         }
 
+        [Fact]
+        public static void Encrypt_Data_DoesNotIncreaseInSize()
+        {
+            byte[] content = new byte[15]; // One short of AES block size boundary
+            ContentInfo contentInfo = new ContentInfo(content);
+            AlgorithmIdentifier identifier = new AlgorithmIdentifier(new Oid(Oids.Aes128));
+            EnvelopedCms ecms = new EnvelopedCms(contentInfo, identifier);
+
+            using (X509Certificate2 cert = Certificates.RSAKeyTransfer1.GetCertificate())
+            {
+                CmsRecipient recipient = new CmsRecipient(cert);
+                ecms.Encrypt(recipient);
+            }
+
+            byte[] encoded = ecms.Encode();
+            EnvelopedCms reDecoded = new EnvelopedCms();
+            reDecoded.Decode(encoded);
+            Assert.Equal(16, reDecoded.ContentInfo.Content.Length);
```

Comment:
Right, 128 bits is only 16 bytes, not 32.  Maybe... I... was in... a 4-bit operating system?  (Math major, excused from arithmetic)

Did this test correctly fail with my broken version?

Reply:
@bartonjs Indeed. With the `EncodeOctetString` restored, test fails with.

```
System.Security.Cryptography.Pkcs.EnvelopedCmsTests.Tests.GeneralTests.Encrypt_Data_DoesNotIncreaseInSize [FAIL]
        Assert.Equal() Failure
        Expected: 16
        Actual:   32
```